SourceGenerators/SourceGeneratorForDeclaredMemberWithAttribute.cs

```diff
@@ -50,31 +59,76 @@ bool HasAttributeType()
             static TDeclarationSyntax GetSyntaxTarget(GeneratorSyntaxContext context, CancellationToken _)
                 => (TDeclarationSyntax)context.Node;
 
-            void OnExecute(SourceProductionContext context, Compilation compilation, ImmutableArray<TDeclarationSyntax> nodes, AnalyzerConfigOptionsProvider options)
+            void OnExecute(
+                SourceProductionContext context,
+                Compilation compilation,
+                ImmutableArray<TDeclarationSyntax> nodes,
+                AnalyzerConfigOptionsProvider options)
             {
                 try
                 {
+                    var unresolvedDependencies = new Queue<CodeDependency>();
```

Comment:
The dependencies here are scoped to a specific attribute. They might be moved to a static level once cross-attribute dependencies are requried.